CMakePresets.json

```diff
@@ -3,23 +3,51 @@
     "configurePresets": [
         {
             "name": "debug",
-            "displayName": "Custom configure preset",
+            "displayName": "debug-windows",
             "description": "Sets Ninja generator, build and install directory",
             "generator": "Ninja",
             "binaryDir": "${sourceDir}/build/${presetName}",
             "cacheVariables": {
                 "CMAKE_BUILD_TYPE": "Debug",
+                "CMAKE_CXX_STANDARD": "20",
+                "CMAKE_INSTALL_PREFIX": "${sourceDir}/out/install/${presetName}",
+                "VCPKG_TARGET_TRIPLET": "x64-windows"
+            },
+            "architecture": {
+                "value": "x64",
+                "strategy": "external"
+            },
+            "toolchainFile": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake",
```

Comment:
So this toolchain requires vcpkg?  Is that dependency documented in the README?  If not you better do that.
Is how to use the presets documented?

Reply:
The system is currently using vcpkg to pull in boost. It's undocumented, and I'd like to get rid of it in favor of fetch content, but not in this PR.